debian/additions/mariadb.cnf

```diff
@@ -1,19 +1,23 @@
-# MariaDB-specific config file.
-# Read by /etc/mysql/my.cnf
-
-[client]
-# Default is Latin1, if you need UTF-8 set this (also in server section)
-#default-character-set = utf8 
+# The MariaDB configuration file
+#
+# The MariaDB/MySQL tools read configuration files in the following order:
+# 1. "/etc/mysql/mariadb.cnf" (this file) to set global defaults,
```

Comment:
Shouldn't the first item in the list be `/etc/my.cnf`?

Reply:
Drafted a "0." item in #1586 but I won't make a PR for adding one line of comments now...